shared/src/native-src/logging.h

```diff
@@ -102,6 +102,15 @@ namespace shared {
         return std::regex_replace(process_name_without_extension, dash_or_space_or_tab, "_");
     }
 
+    static std::string BuildLogFileSuffix()
+    {
+        std::ostringstream oss;
+
+        auto processName = SanitizeProcessName(ToString(GetCurrentProcessName()));
+        oss << "-" << processName << "-" << GetPID();
+        return oss.str();
+    }
```

Comment:
We have code for this in the tracer native library, is it worth re-using some/all of that: 

https://github.com/DataDog/dd-trace-dotnet/blob/master/tracer/src/Datadog.Trace.ClrProfiler.Native/logger_impl.h#L65-L150

Reply:
Good point, this is something I was asking myself. 
I'll check, because if we want to do something clean, we need:
- move the spdlog artifacts in the shared folder (because tracer uses a newer version of spdlog)
- make sure Profiler and Tracer compile with this new change.

That said, I'll change my PR and actually do that.

Reply:
I'll make a larger change: move shared sources/libraries to `shared` folder.
Because today Profiler and Tracer have in common:
- CLR include files
- fmt library
- spdlog library